taxcalc/functions.py

```diff
@@ -1025,25 +1050,20 @@ def F2441(_earned, _fixeic, e59560, MARS, f2441, DCC_c,
         _earned = e59560
 
     if MARS == 2 and puf:
-        if e32890 != 0:
-            c32880 = 0.5 * _earned
-            c32890 = 0.5 * _earned
-        else:
-            c32880 = max(0., e32880)
-            c32890 = max(0., e32890)
-
-    elif MARS != 2:
```

Comment:
`MARS == 2` means " married filing a joint return or Widow(er) with dependent child (surviving spouse)". And in this case we should split `_earned` in presumably an even way.